README.md

```diff
@@ -9,8 +9,8 @@ id
 <!--- @end-badges@ --->
 
 `id` is a Python tool for generating OIDC identities. It can automatically
-detect and produce OIDC credentials on an number of environments, including GitHub Actions
-and Google Cloud.
+detect and produce OIDC credentials on a number of environments, including
+GitHub Actions, GitLab pipelines and Google Cloud.
```

Comment:
Nitpick, nonblocking: maybe link to the "supported environments" section instead of listing a subset here?

```suggestion
detect and produce OIDC credentials in a [number of environments](#supported-environments).
```

Reply:
I'd prefer to keep it as is and highlight the most widely used ones here.